tutorials/generate-events.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+title: Generating Events
+---
+
+Kite applications work with Big Data. `GenerateEvents.java` generates 1-1.5 million random event records, a small amount of realistic Big Data you can use with Kite examples.
+
+Much of the class is devoted to creating random values. The two methods of interest are `run` and `generateRandomEvent`.
+
+The `run` method performs the following tasks:
+* creates a view of the `hive:events` dataset
+* creates a writer instance
+* spends 36 seconds writing random events
+* closes the writer, which stores the results in the `events` dataset.
+
+While the goal is to create random events, if they're _too_ random there won't be anything to aggregate. The `while` loop simulates a user session with random values for `sessionId`, `userId`, and `ip`. It then generates up to 25 random events for that session.
```

Comment:
I don't think people need to understand the code for events generation unless they are interested. Could you move the relevant information about _what_ is happening to the intro, then just cover _how_ in this section?